meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx_git.bb

```diff
@@ -1,12 +1,12 @@
-include recipes-kernel/linux/kmeta-linux-lmp-5.15.y.inc
+include recipes-kernel/linux/kmeta-linux-lmp-6.1.y.inc
 
 # Use Freescale kernel by default
 KERNEL_REPO ?= "git://github.com/Freescale/linux-fslc.git"
 KERNEL_REPO_PROTOCOL ?= "https"
-LINUX_VERSION ?= "5.15.87"
-KERNEL_BRANCH ?= "5.15-2.2.x-imx"
+LINUX_VERSION ?= "6.1.24"
+KERNEL_BRANCH ?= "6.1-1.0.x-imx"
```

Comment:
I think until we can have the other core backported changes in meta-freescale, it might make sense to have this as a separated recipe (6.1 specific) and with default preference to -1.

Reply:
Agree. Done.